cloudflare_logpush: expand set of supported fields and add data streams

efd6 opened the pull request:
<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Bug
- Enhancement
- Breaking change
- Deprecation
-->

## Proposed commit message

<!-- Mandatory
Explain here the changes you made on the PR.

Please explain:

- WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
- WHY:  the rationale/motivation for the changes

This text will be pasted into the squash dialog when the change is committed and will be
a long term historical record of the change to help future contributors understand the
change, please help them by making it clear and comprehensive, they may be you.

If the commit title is adequate to describe both of these things, The text here may be omitted
or replaced with "See title". The title of the PR will be used as the commit message title when
the merge is made and the "See title" marker will be removed if present.

The text here and the PR title will be subject to the PR review process.
-->

See title.

> [!WARNING]
> No test cases are available for the new fields and tests for the new data streams are constructed from documentation without any ground truth.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

<!-- Recommended
Add a checklist of things that are required to be reviewed in order to have the PR approved
-->
- [ ] 

## How to test this PR locally

<!-- Recommended
Explain here how this PR will be tested by the reviewer: commands, dependencies, steps, etc.
-->

## Related issues

<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes #123
- Relates #123
- Requires #123
- Supersedes #123
-->
- Closes #9809
- For #13002

## Screenshots

<!-- Optional
Add here screenshots presenting:
- Kibana UI forms presenting configuration options exposed by the integration
- dashboards with collected metrics or logs
-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/22289)
* Commit: 7b5c0ea3463d1bdb61fc9790343d06e29871632b

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/22289","number":22289,"commit":"7b5c0ea3463d1bdb61fc9790343d06e29871632b"}],"number":22289}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23020)
* Commit: b4e859479800886e2a3b56b99b71f02bca92914e

### History
* :green_heart: [Build #22289](https://buildkite.com/elastic/integrations/builds/22289) succeeded 7b5c0ea3463d1bdb61fc9790343d06e29871632b

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23020","number":23020,"commit":"b4e859479800886e2a3b56b99b71f02bca92914e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/22289","number":22289,"commit":"7b5c0ea3463d1bdb61fc9790343d06e29871632b"}],"number":23020}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23052)
* Commit: 370028209ca57bf0debc3556b19d3c12eb853a1b

### Failed CI Steps
* [Check integrations cloudflare_logpush](https://buildkite.com/elastic/integrations/builds/23052#019568af-e3c9-4b98-9394-7dfe1142b206)

### History
* :green_heart: [Build #23020](https://buildkite.com/elastic/integrations/builds/23020) succeeded b4e859479800886e2a3b56b99b71f02bca92914e
* :green_heart: [Build #22289](https://buildkite.com/elastic/integrations/builds/22289) succeeded 7b5c0ea3463d1bdb61fc9790343d06e29871632b

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23052","number":23052,"commit":"370028209ca57bf0debc3556b19d3c12eb853a1b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23020","number":23020,"commit":"b4e859479800886e2a3b56b99b71f02bca92914e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/22289","number":22289,"commit":"7b5c0ea3463d1bdb61fc9790343d06e29871632b"}],"number":23052}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23055)
* Commit: 40b064118b689e0c80ad37df33cb686432bae3a3

### History
* :broken_heart: [Build #23052](https://buildkite.com/elastic/integrations/builds/23052) failed 370028209ca57bf0debc3556b19d3c12eb853a1b
* :green_heart: [Build #23020](https://buildkite.com/elastic/integrations/builds/23020) succeeded b4e859479800886e2a3b56b99b71f02bca92914e
* :green_heart: [Build #22289](https://buildkite.com/elastic/integrations/builds/22289) succeeded 7b5c0ea3463d1bdb61fc9790343d06e29871632b

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23055","number":23055,"commit":"40b064118b689e0c80ad37df33cb686432bae3a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23052","number":23052,"commit":"370028209ca57bf0debc3556b19d3c12eb853a1b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23020","number":23020,"commit":"b4e859479800886e2a3b56b99b71f02bca92914e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/22289","number":22289,"commit":"7b5c0ea3463d1bdb61fc9790343d06e29871632b"}],"number":23055}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/security-service-integrations (Team:Security-Service Integrations)

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23068)
* Commit: a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e

### History
* :green_heart: [Build #23055](https://buildkite.com/elastic/integrations/builds/23055) succeeded 40b064118b689e0c80ad37df33cb686432bae3a3
* :broken_heart: [Build #23052](https://buildkite.com/elastic/integrations/builds/23052) failed 370028209ca57bf0debc3556b19d3c12eb853a1b
* :green_heart: [Build #23020](https://buildkite.com/elastic/integrations/builds/23020) succeeded b4e859479800886e2a3b56b99b71f02bca92914e
* :green_heart: [Build #22289](https://buildkite.com/elastic/integrations/builds/22289) succeeded 7b5c0ea3463d1bdb61fc9790343d06e29871632b

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23068","number":23068,"commit":"a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23055","number":23055,"commit":"40b064118b689e0c80ad37df33cb686432bae3a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23052","number":23052,"commit":"370028209ca57bf0debc3556b19d3c12eb853a1b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23020","number":23020,"commit":"b4e859479800886e2a3b56b99b71f02bca92914e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/22289","number":22289,"commit":"7b5c0ea3463d1bdb61fc9790343d06e29871632b"}],"number":23068}
buildkite-pr-comment-->

chemamartinez commented on packages/cloudflare_logpush/data_stream/gateway_network/elasticsearch/ingest_pipeline/default.yml:
```diff
@@ -174,9 +174,19 @@ processors:
       copy_from: cloudflare_logpush.gateway_network.source.port
       ignore_empty_value: true
+  - rename:
+      field: json.TransportName
```
Based on [Gateway Network docs](https://developers.cloudflare.com/logs/reference/log-fields/account/gateway_network/#transportprotocol), wouldn't be `TransportProtocol`?

efd6 replied:
I'm not sure where that came from.

chemamartinez commented on packages/cloudflare_logpush/data_stream/gateway_network/elasticsearch/ingest_pipeline/default.yml:
```diff
@@ -174,9 +174,19 @@ processors:
+  - append:
+      field: cloudflare_logpush.gateway_network.transport
+      value: '{{{json.Transport}}}'
+      if: ctx.cloudflare_logpush?.gateway_network?.policy?.name != null && ctx.json?.Policy != ctx.cloudflare_logpush?.gateway_network?.policy?.name
```
I don't understand the relationship here between the transport field and the policy name.

efd6 replied:
This and below were copy-paste error from gateway_dns. Will fix.

chemamartinez commented on packages/cloudflare_logpush/data_stream/gateway_network/elasticsearch/ingest_pipeline/default.yml:
```diff
@@ -174,9 +174,19 @@ processors:
   - rename:
       field: json.Transport
       target_field: cloudflare_logpush.gateway_network.transport
+      if: ctx.cloudflare_logpush?.gateway_network?.policy?.name == null
```
Same.

chemamartinez commented on packages/cloudflare_logpush/data_stream/gateway_network/elasticsearch/ingest_pipeline/default.yml:
```diff
@@ -219,10 +257,34 @@ processors:
+      ignore_missing: true
+  - rename:
+      field: json.TransportProtocol
+      target_field: cloudflare_logpush.gateway_network.transport
```
I'd move this rename processor to be located before this one:

```
  - set:
       field: network.transport
      copy_from: cloudflare_logpush.gateway_network.transport
      ignore_empty_value: true
```

I think the logic with the Transport fields should be:
- Rename `json.TransportProtocol` to `cloudflare_logpush.gateway_network.transport`
- Rename `json.Transport` to `cloudflare_logpush.gateway_network.transport` if empty (deprecated)
- Set `network.transport`

efd6 replied:
Yeah, this is the confusion that was in the two comments above.

chemamartinez commented on packages/cloudflare_logpush/data_stream/network_analytics/elasticsearch/ingest_pipeline/default.yml:
```diff
@@ -608,6 +716,30 @@ processors:
+        - append:
+            field: error.message
+            value: '{{{_ingest.on_failure_message}}}'
   - convert:
```
This `convert` processor should be removed, it has been replaced by the logic above.

chemamartinez commented on packages/cloudflare_logpush/data_stream/page_shield_events/agent/stream/aws-s3.yml.hbs:
```diff
@@ -0,0 +1,145 @@
+{{/if}}
+
+{{! AWS S3 bucket ARN options }}
+{{#unless cloudflare_r2_firewall_event}}
```
```suggestion
{{#unless cloudflare_r2_page_shield_events}}
```

Same for the rest of `firewall_event` references here and in the manifest.

chemamartinez reviewed (commented):
I think we still need to add some fields for the `gateway_http` data stream, the ones that are not linked in the table of the original issue.

Notice that the link to the documentation for the `gateway_http` data stream is pointing to the `gateway_network` dataset so that should be why you couldn't find them.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23377)
* Commit: 42d35d4521d46d12d19910d7bc6cc8ef3e4e5970

### Failed CI Steps
* [Check integrations cloudflare_logpush](https://buildkite.com/elastic/integrations/builds/23377#01959133-590e-4132-9c74-9b3738515a72)

### History
* :green_heart: [Build #23068](https://buildkite.com/elastic/integrations/builds/23068) succeeded a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e
* :green_heart: [Build #23055](https://buildkite.com/elastic/integrations/builds/23055) succeeded 40b064118b689e0c80ad37df33cb686432bae3a3
* :broken_heart: [Build #23052](https://buildkite.com/elastic/integrations/builds/23052) failed 370028209ca57bf0debc3556b19d3c12eb853a1b
* :green_heart: [Build #23020](https://buildkite.com/elastic/integrations/builds/23020) succeeded b4e859479800886e2a3b56b99b71f02bca92914e
* :green_heart: [Build #22289](https://buildkite.com/elastic/integrations/builds/22289) succeeded 7b5c0ea3463d1bdb61fc9790343d06e29871632b

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23377","number":23377,"commit":"42d35d4521d46d12d19910d7bc6cc8ef3e4e5970"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23068","number":23068,"commit":"a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23055","number":23055,"commit":"40b064118b689e0c80ad37df33cb686432bae3a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23052","number":23052,"commit":"370028209ca57bf0debc3556b19d3c12eb853a1b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23020","number":23020,"commit":"b4e859479800886e2a3b56b99b71f02bca92914e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/22289","number":22289,"commit":"7b5c0ea3463d1bdb61fc9790343d06e29871632b"}],"number":23377}
buildkite-pr-comment-->

efd6 commented:
@chemamartinez I marked the missing (now corrected) fields in the table in the issue with a `*` so that it's clear which were added. The link is also fixed — thanks for noticing that.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23379)
* Commit: 546acaf35e84f073b5ef9fd07bff1988f767d3c0

### History
* :broken_heart: [Build #23377](https://buildkite.com/elastic/integrations/builds/23377) failed 42d35d4521d46d12d19910d7bc6cc8ef3e4e5970
* :green_heart: [Build #23068](https://buildkite.com/elastic/integrations/builds/23068) succeeded a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e
* :green_heart: [Build #23055](https://buildkite.com/elastic/integrations/builds/23055) succeeded 40b064118b689e0c80ad37df33cb686432bae3a3
* :broken_heart: [Build #23052](https://buildkite.com/elastic/integrations/builds/23052) failed 370028209ca57bf0debc3556b19d3c12eb853a1b
* :green_heart: [Build #23020](https://buildkite.com/elastic/integrations/builds/23020) succeeded b4e859479800886e2a3b56b99b71f02bca92914e

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23379","number":23379,"commit":"546acaf35e84f073b5ef9fd07bff1988f767d3c0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23377","number":23377,"commit":"42d35d4521d46d12d19910d7bc6cc8ef3e4e5970"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23068","number":23068,"commit":"a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23055","number":23055,"commit":"40b064118b689e0c80ad37df33cb686432bae3a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23052","number":23052,"commit":"370028209ca57bf0debc3556b19d3c12eb853a1b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23020","number":23020,"commit":"b4e859479800886e2a3b56b99b71f02bca92914e"}],"number":23379}
buildkite-pr-comment-->

chemamartinez commented on packages/cloudflare_logpush/data_stream/dlp_forensic_copies/manifest.yml:
```diff
@@ -0,0 +1,261 @@
+      - name: url
+        type: text
+        title: URL
+        description: This option specifies which URL path to accept requests on. Defaults to /cloudflare_logpush/firewall_event.
```
There are some other references of `firewall_event` in the new manifests. Please check these references across the manifests of the new data streams.

chemamartinez reviewed (approved):
One comment left. Apart from that, it looks good!

kcreddy commented on packages/entityanalytics_ad/data_stream/user/_dev/test/pipeline/test-user.json-expected.json:
```diff
@@ -498,4 +498,4 @@
     ]
-}
\ No newline at end of file
+}
```
This change is from another package. Can we remove this?

andrewkroh replied:
Moving this back to unresolved because a `packages/entityanalytics_ad` change is present.

kcreddy replied:
Looks like it got re-introduced by https://github.com/elastic/integrations/pull/12782/commits/b99f3e49cf3ae0a937a6011ec0fafab7ad79a89b

kcreddy commented on packages/cloudflare_logpush/data_stream/page_shield_events/agent/stream/http_endpoint.yml.hbs:
```diff
@@ -0,0 +1,45 @@
+{{processors}}
+{{/if}}
+{{#if enable_request_tracer}}
+tracer.filename: "../../logs/http_endpoint/http-request-trace-*.ndjson"
```
Since min version is `8.16.5`, maybe we could add [`tracer.enabled`](https://www.elastic.co/guide/en/beats/filebeat/8.15/filebeat-input-http_endpoint.html#_tracer_enabled_3)

efd6 replied:
Good catch.

kcreddy commented on packages/cloudflare_logpush/data_stream/page_shield_events/_dev/test/pipeline/test-pipeline-page-shield-events.log-expected.json:
```diff
@@ -0,0 +1,49 @@
+                ]
+            },
+            "host": {
+                "name": "hostymchost.face"
```
Could be added to related.hosts

kcreddy commented on packages/cloudflare_logpush/data_stream/dlp_forensic_copies/_dev/test/pipeline/test-pipeline-dlp-forensic-copies.log-expected.json:
```diff
@@ -0,0 +1,39 @@
+                    },
+                    "payload": "Tm90aGluZyB0byBzZWUgaGVyZS4gTW92ZSBhbG9uZy4K",
+                    "phase": "request",
+                    "triggered_rule_id": "9"
```
Could be copied into `rule.id`

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23431)
* Commit: 7928300359be82912240b6bce9bbc3a6cbdd304f

### History
* :green_heart: [Build #23379](https://buildkite.com/elastic/integrations/builds/23379) succeeded 546acaf35e84f073b5ef9fd07bff1988f767d3c0
* :broken_heart: [Build #23377](https://buildkite.com/elastic/integrations/builds/23377) failed 42d35d4521d46d12d19910d7bc6cc8ef3e4e5970
* :green_heart: [Build #23068](https://buildkite.com/elastic/integrations/builds/23068) succeeded a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e
* :green_heart: [Build #23055](https://buildkite.com/elastic/integrations/builds/23055) succeeded 40b064118b689e0c80ad37df33cb686432bae3a3
* :broken_heart: [Build #23052](https://buildkite.com/elastic/integrations/builds/23052) failed 370028209ca57bf0debc3556b19d3c12eb853a1b
* :green_heart: [Build #23020](https://buildkite.com/elastic/integrations/builds/23020) succeeded b4e859479800886e2a3b56b99b71f02bca92914e

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23431","number":23431,"commit":"7928300359be82912240b6bce9bbc3a6cbdd304f"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23379","number":23379,"commit":"546acaf35e84f073b5ef9fd07bff1988f767d3c0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23377","number":23377,"commit":"42d35d4521d46d12d19910d7bc6cc8ef3e4e5970"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23068","number":23068,"commit":"a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23055","number":23055,"commit":"40b064118b689e0c80ad37df33cb686432bae3a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23052","number":23052,"commit":"370028209ca57bf0debc3556b19d3c12eb853a1b"}],"number":23431}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/23839)
* Commit: cce278da5f053fca92287bf60781e8f6249f3d34

### History
* :green_heart: [Build #23431](https://buildkite.com/elastic/integrations/builds/23431) succeeded 7928300359be82912240b6bce9bbc3a6cbdd304f
* :green_heart: [Build #23379](https://buildkite.com/elastic/integrations/builds/23379) succeeded 546acaf35e84f073b5ef9fd07bff1988f767d3c0
* :broken_heart: [Build #23377](https://buildkite.com/elastic/integrations/builds/23377) failed 42d35d4521d46d12d19910d7bc6cc8ef3e4e5970
* :green_heart: [Build #23068](https://buildkite.com/elastic/integrations/builds/23068) succeeded a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e
* :green_heart: [Build #23055](https://buildkite.com/elastic/integrations/builds/23055) succeeded 40b064118b689e0c80ad37df33cb686432bae3a3
* :broken_heart: [Build #23052](https://buildkite.com/elastic/integrations/builds/23052) failed 370028209ca57bf0debc3556b19d3c12eb853a1b

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23839","number":23839,"commit":"cce278da5f053fca92287bf60781e8f6249f3d34"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23431","number":23431,"commit":"7928300359be82912240b6bce9bbc3a6cbdd304f"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23379","number":23379,"commit":"546acaf35e84f073b5ef9fd07bff1988f767d3c0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23377","number":23377,"commit":"42d35d4521d46d12d19910d7bc6cc8ef3e4e5970"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23068","number":23068,"commit":"a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23055","number":23055,"commit":"40b064118b689e0c80ad37df33cb686432bae3a3"}],"number":23839}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/24070)
* Commit: 35543da44ef81d35da00e8fdb52491711806b7f6

### History
* :green_heart: [Build #23839](https://buildkite.com/elastic/integrations/builds/23839) succeeded cce278da5f053fca92287bf60781e8f6249f3d34
* :green_heart: [Build #23431](https://buildkite.com/elastic/integrations/builds/23431) succeeded 7928300359be82912240b6bce9bbc3a6cbdd304f
* :green_heart: [Build #23379](https://buildkite.com/elastic/integrations/builds/23379) succeeded 546acaf35e84f073b5ef9fd07bff1988f767d3c0
* :broken_heart: [Build #23377](https://buildkite.com/elastic/integrations/builds/23377) failed 42d35d4521d46d12d19910d7bc6cc8ef3e4e5970
* :green_heart: [Build #23068](https://buildkite.com/elastic/integrations/builds/23068) succeeded a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e
* :green_heart: [Build #23055](https://buildkite.com/elastic/integrations/builds/23055) succeeded 40b064118b689e0c80ad37df33cb686432bae3a3

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24070","number":24070,"commit":"35543da44ef81d35da00e8fdb52491711806b7f6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23839","number":23839,"commit":"cce278da5f053fca92287bf60781e8f6249f3d34"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23431","number":23431,"commit":"7928300359be82912240b6bce9bbc3a6cbdd304f"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23379","number":23379,"commit":"546acaf35e84f073b5ef9fd07bff1988f767d3c0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23377","number":23377,"commit":"42d35d4521d46d12d19910d7bc6cc8ef3e4e5970"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23068","number":23068,"commit":"a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e"}],"number":24070}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/24200)
* Commit: 64e370ab9721c49984f14f99e23fc9a80eeb35af

### History
* :green_heart: [Build #24070](https://buildkite.com/elastic/integrations/builds/24070) succeeded 35543da44ef81d35da00e8fdb52491711806b7f6
* :green_heart: [Build #23839](https://buildkite.com/elastic/integrations/builds/23839) succeeded cce278da5f053fca92287bf60781e8f6249f3d34
* :green_heart: [Build #23431](https://buildkite.com/elastic/integrations/builds/23431) succeeded 7928300359be82912240b6bce9bbc3a6cbdd304f
* :green_heart: [Build #23379](https://buildkite.com/elastic/integrations/builds/23379) succeeded 546acaf35e84f073b5ef9fd07bff1988f767d3c0
* :broken_heart: [Build #23377](https://buildkite.com/elastic/integrations/builds/23377) failed 42d35d4521d46d12d19910d7bc6cc8ef3e4e5970
* :green_heart: [Build #23068](https://buildkite.com/elastic/integrations/builds/23068) succeeded a92b51b71f3fb4f9ebc2edaff9ffde56de253f4e

cc @efd6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24200","number":24200,"commit":"64e370ab9721c49984f14f99e23fc9a80eeb35af"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/24070","number":24070,"commit":"35543da44ef81d35da00e8fdb52491711806b7f6"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23839","number":23839,"commit":"cce278da5f053fca92287bf60781e8f6249f3d34"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23431","number":23431,"commit":"7928300359be82912240b6bce9bbc3a6cbdd304f"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23379","number":23379,"commit":"546acaf35e84f073b5ef9fd07bff1988f767d3c0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/23377","number":23377,"commit":"42d35d4521d46d12d19910d7bc6cc8ef3e4e5970"}],"number":24200}
buildkite-pr-comment-->
